README.md

```diff
@@ -133,8 +133,10 @@ something has changed.
 
 1. Run `neqo-server` via `cargo run --bin neqo-server -- 'localhost:12345' --db ./test-fixture/db`.
 2. On Firefox, set `about:config` preferences:
-  - `network.http.http3.alt-svc-mapping-for-testing` to `localhost;h3=":12345"`
-  - `network.http.http3.disable_when_third_party_roots_found` to `false`
+
+- `network.http.http3.alt-svc-mapping-for-testing` to `localhost;h3=":12345"`
```

Comment:
This is a sub-bullet of (2). Maybe it needs further indentation for a formatter to realize it?